scalafix-reflect/src/main/scala/scalafix/internal/reflect/ClasspathOps.scala

```diff
@@ -29,15 +29,16 @@ object ClasspathOps {
   }
 
   def thisClassLoader: URLClassLoader =
-    this.getClass.getClassLoader.asInstanceOf[URLClassLoader]
+    new URLClassLoader(Array.empty, this.getClass().getClassLoader)
```

Comment:
We can pass `this.getClass.getClassLoader` to `getURLs` from the method here https://github.com/scalameta/metals/blob/2eb7a239a2e4e379c794ae2d66c6977ac9014bda/mtags/src/main/scala/scala/meta/internal/mtags/ClasspathLoader.scala#L14-L52

That should preserve the behavior on JDK 8 and 9+

Reply:
Thanks! Fixed